docfx_yaml/markdown_utils.py

```diff
@@ -280,7 +315,13 @@ def move_markdown_pages(
 
         "reference.md", # Reference docs overlap with Overview. Will try and incorporate this in later.
                         # See https://github.com/googleapis/sphinx-docfx-yaml/issues/106.
-        "summary_overview.md",  # Already included in the TOC.
+        "overview_content.md",  # Will be included in the summary_overview page.
+    ]
+
+    # Use this to move the page but do not add them in the TOC.
```

Comment:
Ah, this is to (1) move the file from `docs/summary_overview.md` to `output_location/summary_overview.md`.
In the process we also (2) include it in `app.env.moved_markdown_pages` and (3) include it in the TOC as well.

We do need the summary_overview page in the output location for it to be used. However we don't need it in (2) or (3) because triggering either conditions will either (a) remove the page, or (b) include it _again_ in the TOC.